README.md

```diff
@@ -234,5 +234,5 @@ Supported Platforms:
 Supported Hardware:
 
 - NVIDIA GPUs (Compute Capability 8.0+)
-- AMD GPUs (ROCm 5.2+)
+- AMD GPUs (ROCm 6.2+)
```

Comment:
So rocm 5.x is no longer supported?

Reply:
Due to the switch to cov5 `test_optimize_thread_locality` and some `device_prints` (the new test) will cause segfaults on 6.1 and 6.0, I haven't tested 5.x versions.

Reply:
I think we need to chat a bit regarding when to land this then, given it's a breaking change.